app/assets/stylesheets/components/header.scss

```
@@ -137,29 +139,39 @@
 
 .skip-content-link {
   position: absolute;
-  top: 0;
   left: 50%;
-  transform: translate(-50%, -100px);
   background: var(--body-color);
   color: var(--body-color-inverted);
   z-index: 9999;
   padding: var(--su-4);
   pointer-events: none;
   opacity: 0;
   border-radius: var(--radius);
-  outline: 0;
+  height: var(-header-height);
+  transform: translateY(-10em);
+  transition: transform 0.2s ease-in-out;
 
-  &:focus {
+  &:focus,
+  &:focus-within {
     pointer-events: auto;
     opacity: 1;
-    transform: translate(-50%, var(--header-height) / 2);
+    transform: translateY(0em);
+    -webkit-transition: top 0.1s ease-in;
+    transition: top 0.1s ease-in;
+  }
+
+  @media (prefers-reduced-motion: reduce) {
+    &.skip-content-link {
+      transition-duration: 0.0001ms !important;
```

➕ **1** for adding `prefers-reduced-motion` 👏🏻

I'm not positive, but is the `!important` necessary? Doesn't `&.skip-content-link` translate to `.skip-content-link.skip-content-link` which would would have a higher specificity?

---

@nickytonline `!important` overrides any other declarations. Techncally, `!important` has nothing to do with specificity, it interact directly with it. As much as it is widely known not to use it, I did, although I will leave this up to you all as to whether you want to implement that or not.

---

I think we can omit the `!important`.

```suggestion
      transition-duration: 0.0001ms;
```